src/index.js

```diff
@@ -47,10 +49,7 @@ const edgeCorePlugins = {
   transfer: makeTransferPlugin
 }
 
-if (
-  typeof window !== 'undefined' &&
-  typeof window.addEdgeCorePlugins === 'function'
-) {
+if (typeof window !== 'undefined' && typeof window.addEdgeCorePlugins === 'function') {
```

Comment:
This line should be the way it was, before the weird prettier file messed it up.